api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceUserRoleDao.java

```diff
@@ -0,0 +1,11 @@
+package org.pmiops.workbench.db.dao;
+
+import org.pmiops.workbench.db.model.User;
+import org.pmiops.workbench.db.model.Workspace;
+import org.pmiops.workbench.db.model.WorkspaceUserRole;
+import org.springframework.data.repository.CrudRepository;
+
+public interface WorkspaceUserRoleDao extends CrudRepository<WorkspaceUserRole, Long> {
+   WorkspaceUserRole findWorkspaceUserRolesByWorkspaceAndUser(Workspace workspace, User user);
+}
+
```

Comment:
Do you need a whole new DAO for this? It seems like this query should be in `WorkspaceServiceImpl`